gridsome/app/components/Link.js

```diff
@@ -27,6 +28,18 @@ export default {
       return h('a', data, children)
     }
 
+    
+    const isExternalLinks = string => {
+      if(string === config.siteUrl) return false;
```

Comment:
```suggestion
      if(string.substring(0, config.siteUrl.length) === config.siteUrl) return false
```
Checks if `href` starts with the `siteUrl` to include all pages within `siteUrl`.

Reply:
Since we've got access to newer JS features here, you might consider using [`String.startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith):

```js
if (string.startsWith(config.siteUrl)) return false
```

Reply:
@mikemcbride 
Thank you for your kind advice :)
I don't know this method.
I rewrote the code!

Reply:
Yes, `startsWith` is better. I'm updating the babel config in core to make it work in IE :)